docs-src/modules/developing/pages/queries.adoc

```diff
@@ -2,6 +2,51 @@
 
 The following sections will help with writing queries.
 
+== Final vs Historical Results
+
+Every query can be computed in two different ways: final and historical.
```

Comment:
Maybe:

Every query produces a timeline which may be returned in two different ways -- the final results (at a specific time) or all historic results.
The "result behavior" configures which results are produced.
Queries for historic results return the full history of how the values changed over time for each entity.
Queries for final results return the latest result for each entity at the specified time (default is after all events have been processed).


Specifically, it feels a bit more direct to focus on "this is an option" and "results" rather than "final queries ... final results" (eg., this seems to introduce the concept of final queries and relate it to final results), which seems to muddy the water.

I think we should *just* have "final results" and "historic results", and never mention "final queries" or "historic queries"